.github/workflows/api-ts-publish.yml

```diff
@@ -33,12 +33,12 @@ jobs:
       - name: Upgrade /web
         working-directory: web
         run: |
-          export VERSION=`node -e 'console.log(require("../gen-ts-api/package.json").version)'`
+          export VERSION=`node -e 'console.log(require("../../gen-ts-api/package.json").version)'`
```

Comment:
I dont think this is correct as the web package json is still in the same directory